projects/addon-editor/components/editor-new/editor-new.component.ts

```diff
@@ -116,6 +116,17 @@ export class TuiEditorNewComponent
         this.updateFocused(active);
     }
 
+    /**
+     * @note:
+     * Editor invalidates html attributes inside
+     * and overwrites again html content
+     */
+    onStateChange(): void {
+        if (!!this.editor?.html && this.editor.html !== this.value) {
```

Comment:
Is there really a case when `stateChange` causes content change?

Reply:
Yes, e2e tests therefore fell